01-intro-pytorch/model.py

```diff
@@ -0,0 +1,67 @@
+import torch
+from torch import nn
+from torch.autograd import Variable
+
+class BoW(torch.nn.Module):
+    def __init__(self, nwords, ntags):
+        super(BoW, self).__init__()
+
+        """ variables """
+        type = torch.FloatTensor
+        use_cuda = torch.cuda.is_available()
+
+        if use_cuda:
+            type = torch.cuda.FloatTensor
+
+        self.bias = Variable(torch.zeros(ntags),
+                             requires_grad=True).type(type)
+
+        """ layers """
+        self.embedding = nn.Embedding(nwords, ntags)
```

Comment:
I think it's just a good practice to initialize the parameters in pytorch, specially the embedding matrix.

Reply:
Thanks @MysteryVaibhav, I have fixed the problem. The accuracies are also better and at par with dynet. The dynet defualt initialization is Glorot initialization (Glorot, X. & Bengio, Y. (2010)). So I used the same.